src/cmd_line/commands/tab.ts

```diff
@@ -142,7 +145,7 @@ export class TabCommand extends ExCommand {
         if (this.arguments.count !== undefined && this.arguments.count >= 0) {
           await vscode.commands.executeCommand(
             'workbench.action.openEditorAtIndex',
-            this.arguments.count
+            this.arguments.count - 1
```

Comment:
Looks like `{count}gt` uses this same code path, so I think we need to adjust the offset [here](https://github.com/VSCodeVim/Vim/blob/1fc416d0513144676d0484ce985eb4b6a07ccfc6/src/actions/commands/actions.ts#L1319) to avoid breaking it